### PR TITLE
minor changes for final JOSS publication

### PR DESCRIPTION
**Pull request by danielskatz**

### CODE REVIEW

File: paper.bib [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @article{boltzmann1868studien,
-  title={Studien über das {G}leichgewicht der lebendigen {K}raft zwischen bewegten materiellen {P}unkten},
+  title={Studien über das Gleichgewicht der lebendigen Kraft zwischen bewegten materiellen Punkten},
```
Comment (mstimberg) [inline paper.bib:2]: Just a comment on this: In German, these words cannot be spelled with a lower case initial (regardless of their position in a sentence, sentence case vs. title case, etc.), so I think they need to be escaped with `{…}` here.

Comment (danielskatz) [reply paper.bib:2]: ok, I'll unchange them. I didn't see them as proper names/places, but I don't know German rules

Comment (danielskatz) [inline paper.bib:2]:
```suggestion
  title={Studien über das {G}leichgewicht der lebendigen {K}raft zwischen bewegten materiellen {P}unkten},
```

File: paper.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ CBX methods have been successfully applied and extended to several different set
 
 # Statement of need
 
-In general, very few implementations of CBO already exist, and none have been designed with the generality of other CBX methods in mind. We summarise here the related software:
+In general, very few implementations of CBO already exist, and none have been designed with the generality of other CBX methods in mind. We summarise the related software here:
```
Comment (danielskatz) [inline paper.md:119]:
```suggestion
In general, very few implementations of CBO already exist, and none have been designed with the generality of other CBX methods in mind. Here, we summarise the related software:
```

File: paper.bib (hunk 1 above)
Comment (mstimberg) [reply paper.bib]: Yes, this is a particularity of German grammar – *all* nouns (not just names, places, etc.) are capitalized.

**Pull request merged**